app/adapters/application.js

```diff
@@ -3,15 +3,21 @@ import DS from 'ember-data';
 import ENV from '../config/environment';
 
 export default DS.RESTAdapter.extend(DS.BuildURLMixin, {
+    session: Ember.inject.service(),
     namespace: 'api/v2',
     host: ENV.apiBaseUrl,
     headers: {
         Accept: 'application/json'
     },
     ajax(url, method, hash) {
+        // add trailing slash
+        url.replace(/([^\/])(\?|$)/, '$1/$2');
```

Comment:
`replace` returns a new copy of the string.
`url = url.replace(/([^\/])(\?|$)/, '$1/$2');`